sdk/ml/azure-ai-ml/azure/ai/ml/entities/_compute/compute_instance.py

```diff
@@ -432,6 +438,12 @@ def _load_from_rest(cls, rest_obj: ComputeResource) -> "ComputeInstance":
             if (prop.properties and prop.properties.enable_node_public_ip is not None)
             else True,
             custom_applications=custom_applications,
+            enable_sso=prop.properties.enable_sso
+            if (prop.properties and prop.properties.enable_sso is not None)
+            else True,
+            enable_root_access=prop.properties.enable_root_access
+            if (prop.properties and prop.properties.enable_root_access is not None)
+            else True,
```

Comment:
Have you confirmed, these both will be true by default?

Reply:
Confirmed with compute team that they should be true by default